platforms/jvm/code-quality/src/main/java/org/gradle/api/plugins/quality/PmdExtension.java

```diff
@@ -116,7 +116,7 @@ public void setTargetJdk(TargetJdk targetJdk) {
     /**
      * The maximum number of failures to allow before stopping the build.
      *
-     * If <pre>ignoreFailures</pre> is set, this is ignored and no limit is enforced.
+     * If {@code ignoreFailures} is set, this is ignored and no limit is enforced.
```

Comment:
![image](https://github.com/user-attachments/assets/608d6d97-5301-4120-9539-2905a91e546c)

`<pre>` introduces line breaks, which prevents the single-line description from rendering correctly.